### PR TITLE
add: min-distance for filter tracks

### PR DESCRIPTION
**Pull request by yihong0618**
This `pr` add  min-distance for filter tracks with distance,  because I have a lot of tracks I want to generate grid only more the 10km tracks I think that will be easy.
Also add `python -m venv` in README, and fix the year in the footer.
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,7 +146,7 @@ We currently support
 ## Setup
 1. Clone the repository: `git clone https://github.com/flopp/GpxTrackPoster.git`
 2. `cd GpxTrackPoster`
-3. Create virtualenv: `virtualenv -p /usr/bin/python3 venv`
+3. Create virtualenv: `virtualenv -p /usr/bin/python3 venv` or `python -m venv venv`
```
Comment (Djailla) [inline README.md:149]: Prefer `python3 -m venv venv`

File: gpxtrackposter/cli.py [hunk 2]
```diff
@@ -246,6 +257,7 @@ def main():
         raise ParameterError(f"Bad year range: {args.year}.")
 
     loader.special_file_names = args.special
+    loader.min_length = args.min_distance * 1000
```
Comment (Djailla) [inline gpxtrackposter/cli.py:260]: Where is it used ?

Comment (yihong0618) [reply gpxtrackposter/cli.py:260]: cli

**Comment by flopp** [comment]
Is the actual filtering missing?

**Comment by yihong0618** [comment]
> Is the actual filtering missing?

I think old one only filter distance < 1km, I make it configable, did I missed something?

**Comment by flopp** [comment]
> > Is the actual filtering missing?
> 
> I think old one only filter distance < 1km, I make it configable, did I missed something?

Ah, I see.

**Pull request merged**